quark/core/quark.py

```diff
@@ -214,6 +216,24 @@ def _evaluate_method(self, method) -> List[List[str]]:
             # for the case of MUTF8String
             instruction = [str(x) for x in instruction]
 
+            # Skip the bytecode that is not analyzed.
+            # e.g. invoke-virtual method+xxxx .
+            if (isinstance(bytecode_obj.parameter, str) and
+                "+" in bytecode_obj.parameter):
+               continue
```

Comment:
We should check if it is an invoke-kind bytecode, otherwise other bytecodes like `const-string v0 "a string with +"` will be ignored.

Reply:
Done.